style.css

```diff
@@ -0,0 +1,423 @@
+/* stylelint-disable custom-property-empty-line-before */
+
+* {
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+  padding: 0;
+  margin: 0;
+  --default: #fff;
+  --color-one: #6070ff;
+  --color-two: #396df2;
+  --color-three: #344563;
+  --color-four: #396df2;
```

Comment:
- [ ] Some sections in the mobile version still need to be improved. Kindly work on that and your project will be ok :heavy_check_mark: 
## Your design in mobile version
![banstein github io_Portfolio-Desktop_(iPhone 6_7_8 Plus)](https://user-images.githubusercontent.com/65068771/144718467-07991007-8a44-4532-97a9-9dc71e3f11f2.png)